scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -546,15 +554,71 @@ function scr_initialize_custom() {
 	variable_struct_set(role_spawn_buffs,role[100,10],0);
 	variable_struct_set(role_spawn_buffs,role[100,9],0);
 	variable_struct_set(role_spawn_buffs,role[100,12],0);
-
-	i=-1;repeat(21){i+=1;
-	    race[100,i]=obj_creation.race[100,i];
-	    role[100,i]=obj_creation.role[100,i];
-	    wep1[100,i]=obj_creation.wep1[100,i];
-	    wep2[100,i]=obj_creation.wep2[100,i];
-	    armour[100,i]=obj_creation.armour[100,i];
-	    gear[100,i]=obj_creation.gear[100,i];
-	    mobi[100,i]=obj_creation.mobi[100,i];
+	
+	squad_types = {};
+	var st = { 
+		"command_squad" :[
+				[role[100,5], {"max":1,"min":1}],		//captain
+				[role[100,7], {"max":1,"min":1}],		//company_champion
+				[role[100,15], {"max":1,"min":1}],		//Apothecary
+				["Standard Bearer" , {"max":1,"min":1}],//standard bearer
+				[role[100,3] , {"max":5,"min":0}],		//veterans
+			],
+			"terminator_squad": [
+				[role[100,8], {"max":9,"min":3}],

```

Comment:
what is role 8? please add more comments, like above for captains etc.
In the future we can refactor the roles to use enums or strings.

Reply:
will comment